src/drive/ducks/files/FileOpenerExternal.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
```

Comment:
We renamed FileOpener into FileOpenerExternal since it is either used in  : 

* an intent handler (aka service)
* via cozydrive://

Maybe @drazik you could add a docstring at the top of the file explaining this ?